.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to the Vonage for Visual Studio Code extension?
```

Comment:
What's Vonage/Visual Studio Code? Is this copy-paste from some other template?

Reply:
@MichaelJolley I think we caught another one.